### DISCUSSION TITLE
Papoo Installation in einer lokalen XAMPP Umgebung

### DISCUSSION THREAD

**Discussion by ennum** [PapooSoftware/PapooCMS#8] [Q&A] [answered]
Hallo,
ich habe XAMPP 7.4 auf einem USB Stick installiert und möchte darauf eine Testumgebung für Papoo (Version 2021.07.20) einrichten. Eine neue Datenbank wurde mit phpmyadmin angelegt und das Setup von Papoo gestartet.
Beim Testen der Verbindung zur Datenbank kommt folgende Meldung:

"Notice: Trying to access array offset on value of type null in G:\xampp\htdocs\papoo\lib\ez_sql_mysqli.php on line 264
Notice: Undefined index: dbp in G:\xampp\htdocs\papoo\lib\ez_sql_mysqli.php on line 264"

Wie kann ich das beheben?
Danke für die Hilfe.

Viele Grüße
Enrico

**Comment by ah-papoo** [comment, marked as answer]
This described issue already got fixed in the current version that is not released yet, please override the file with the current version from [here](https://github.com/PapooSoftware/PapooCMS/blob/main/lib/ez_sql_mysqli.php) and try again.
